index.bs

```diff
@@ -1076,6 +1076,19 @@ with a [{{NoInterfaceObject}}] [=extended attribute=],
 and [=callback interfaces=] which declare [=constants=]
 must be annotated with an [{{Exposed}}] [=extended attribute=].
 
+<div algorithm>
+
+The <dfn>qualified name</dfn> of an [=interface=] |I| is defined as follows:
+
+1.  Let |interface| be the [=identifier=] of |I|.
+1.  If |I| has a [{{LegacyNamespace}}] [=extended attribute=], then:
+    1.  Let |namespace| be the identifier argument of the [{{LegacyNamespace}}]
+        [=extended attribute=].
+    1.  Return the concatenation of |namespace|, ".", and |interface|.
```

Comment:
Maybe link [infra concatenate](https://infra.spec.whatwg.org/#string-concatenate)?

Reply:
I always forget that exists. Thanks.